src/ua_securechannel.c

```diff
@@ -222,10 +210,9 @@ hideBytesAsym(UA_SecureChannel *const channel, UA_Byte **const buf_start,
        channel->securityMode == UA_MESSAGESECURITYMODE_SIGNANDENCRYPT) {
         *buf_end -= securityPolicy->asymmetricModule.cryptoModule.
             getLocalSignatureSize(securityPolicy, channel->channelContext);
-        *buf_end -= 2; // padding byte and extraPadding byte
+        *buf_end -= 1; // padding byte and extraPadding byte
```

Comment:
Why 1 byte? We need to hide one byte for the padding byte and one for a potential extra padding byte.

Reply:
fixed